### DISCUSSION TITLE
looping A/B repeat

### DISCUSSION THREAD

**Discussion by Mannie3** [smplayer-dev/smplayer#810] [Ideas]
Unless I'm doing it wrong, I really miss when I start an A/B loop it doesn't automatically loop that section. The video carries on past the loop point and I need to jump back in time to before the start of the loop before it will engage the A/B section. I was hoping that as soon as I select point B it starts looping from point A.

**Comment by smplayer-dev** [comment]
You also need to turn on "repeat" in the A-B section menu.

**Reply by Mannie3** [reply]
yes I've done that, but it does not loop until you go back to a time **before** the start of the loop (A), rather than starting to loop as soon as you set the end of the loop (B), even if the repeat function is already switched on.
